dashboard/src/components/Connect.tsx

```diff
@@ -63,7 +67,7 @@ export default function Connect({ closable }: { closable: boolean }) {
         }
       }}
       backEnabled={backEnabled}
-      closeable={closable}
+      closeable={closeButtonEnabled}
       onClose={() => {
```

Comment:
if I'm not wrong this function needs to be updated too (like on `onNext`)! Since it is a modal I expect it to be closable also if you click outside the modal.